Nodejs/Product/Nodejs/Intellisense/IntellisenseController.cs

```diff
@@ -193,6 +193,13 @@ internal sealed class IntellisenseController : IIntellisenseController, IOleComm
                             UpdateCurrentParameter();
                         }
                         break;
+                    default:
+                        if (IsIdentifierFirstChar(ch) && _activeSession == null

```

Comment:
Did you test this when typing in comments, string literals, etc...? I don't see any filtering here on context (e.g. see some of the checks above like on line 169). It's super annoying when the completion list keeps appears when you're typing in places you don't want it (like those just listed). I've seen those bugs before (I may have written a couple of them :-/ )